data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+		event "temporary outfitter removed"
+		conversation
+			`Your exotic new ship draws a lot of attention from the dockworkers, probably even more so due to the unprecedentedly large cannon. Albert and the Hai archaeologists spend most of the day examining your ship in more detail and recovering the corpses now that it's in a proper spaceport and they can access more complex equipment. Everything is completed in one particularly busy afternoon, and as night falls, the immense scale of the inside of the ship allows more floors to be installed, giving you even more living space.`
+			`	You go back to the same park as before with Albert and the three Hai in partial disguise, ice creams in hand and paws to commemorate the new acquisition.`
+			`	"That is one impressive ship," you mention.`
+			`	"True," Desyo says. "I've never seen a cannon that big before."`
+			choice
+				`	"That's what she said."`
+				`	"I wonder how it works."`
+					goto works
+			`	Desyo doesn't understand, but Eyama bursts into laughter. "Classic human joke. You see, it implies-"`
+			`	"God, don't explain it," Albert interrupts. "That was terrible. She's right though, that isn't something you see every day. Do you have any idea how it works?"`
+			label works
+			`	Rakei, who is more knowledgeable in this area, responds. "It's a fusion-pumped gamma-ray laser, or GRASER for short. The device is simple in concept, but the cost involved and the proportion of everything makes it impractical to develop, not to mention the way it works would probably be considered a war crime in many civilizations, thus making R&D not worth it. It's essentially using a nuclear flash to power a laser beam that penetrates and melts everything. Fascinating, but I'm hoping you'll only use it for peaceful purposes."`
+			`	Eyama adds, "I've been calling it the Dragonflame cannon. I think it's appropriate, even though it's technically not a flame in the chemical sense. The dragons of human legends could breathe fire, and although it's unlikely the Sheragi could actually do that, this cannon more than makes up for it."`
+			choice
+				`	"Do you think I could get in trouble by using it?"`
+					goto trouble
+				`	"It's not going to blow up my own ship, is it?"`
+			`	Rakei laughs. "Don't worry, the Sheragi did their homework. The simple materials they used means the gun can be maintained and repaired, so it continues to be structurally strong. Many of the components are active anyway, and they basically only exist for the short amount of time that the GRASER takes to fire."`
```

Comment:
Maybe replace `gun` with `weapon`? Gun seems a bit of an understatement for something this massive.